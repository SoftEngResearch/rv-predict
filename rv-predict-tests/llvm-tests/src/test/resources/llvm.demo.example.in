<gid:0;tid:1;id:4865423;addr:0;value:0;type:INVOKE_METHOD;fn:__libc_start_main;file:/build/buildd/glibc-2.21/csu/libc-start.c;line:289>
<gid:1;tid:1;id:4479204;addr:2;value:0;type:START;fn:__interceptor_pthread_create;file:tsan/rtl/tsan_interceptors.cc;line:859>
<gid:2;tid:2;id:4865297;addr:0;value:0;type:INVOKE_METHOD;fn:__tsan_thread_start_func;file:/home/traian/Documente/rv/rv-predict-llvm/projects/compiler-rt/lib/tsan/rtl/tsan_interceptors.cc;line:849>
<gid:3;tid:1;id:4383649;addr:21676896;value:0;type:WRITE_LOCK;fn:pthread_mutex_lock;file:tsan/../sanitizer_common/sanitizer_common_interceptors.inc;line:3103>
<gid:4;tid:1;id:4383649;addr:21676896;value:2;type:READ;fn:pthread_mutex_lock;file:tsan/../sanitizer_common/sanitizer_common_interceptors.inc;line:3103>
<gid:5;tid:1;id:4865467;addr:21676936;value:0;type:WRITE;fn:main;file:demo/example.c;line:18>
<gid:6;tid:1;id:4363844;addr:21676896;value:0;type:WRITE_UNLOCK;fn:__interceptor_pthread_mutex_unlock;file:tsan/../sanitizer_common/sanitizer_common_interceptors.inc;line:3114>
<gid:7;tid:1;id:4363844;addr:21676896;value:2;type:READ;fn:__interceptor_pthread_mutex_unlock;file:tsan/../sanitizer_common/sanitizer_common_interceptors.inc;line:3114>
<gid:8;tid:1;id:4865495;addr:140727993339800;value:139916889552640;type:READ;fn:main;file:demo/example.c;line:20>
<gid:9;tid:2;id:4383649;addr:21676896;value:0;type:WRITE_LOCK;fn:pthread_mutex_lock;file:tsan/../sanitizer_common/sanitizer_common_interceptors.inc;line:3103>
<gid:10;tid:2;id:4383649;addr:21676896;value:2;type:READ;fn:pthread_mutex_lock;file:tsan/../sanitizer_common/sanitizer_common_interceptors.inc;line:3103>
<gid:11;tid:2;id:4865328;addr:21676936;value:1;type:WRITE;fn:Thread1;file:demo/example.c;line:8>
<gid:12;tid:2;id:4363844;addr:21676896;value:0;type:WRITE_UNLOCK;fn:__interceptor_pthread_mutex_unlock;file:tsan/../sanitizer_common/sanitizer_common_interceptors.inc;line:3114>
<gid:13;tid:2;id:4363844;addr:21676896;value:2;type:READ;fn:__interceptor_pthread_mutex_unlock;file:tsan/../sanitizer_common/sanitizer_common_interceptors.inc;line:3114>
<gid:14;tid:2;id:4865354;addr:21676936;value:1;type:READ;fn:Thread1;file:demo/example.c;line:10>
<gid:15;tid:2;id:4865297;addr:0;value:0;type:FINISH_METHOD;fn:__tsan_thread_start_func;file:/home/traian/Documente/rv/rv-predict-llvm/projects/compiler-rt/lib/tsan/rtl/tsan_interceptors.cc;line:849>
<gid:16;tid:1;id:4488658;addr:2;value:0;type:JOIN;fn:__interceptor_pthread_join;file:tsan/rtl/tsan_interceptors.cc;line:914>
<gid:17;tid:1;id:4865423;addr:0;value:0;type:FINISH_METHOD;fn:__libc_start_main;file:/build/buildd/glibc-2.21/csu/libc-start.c;line:289>

1
