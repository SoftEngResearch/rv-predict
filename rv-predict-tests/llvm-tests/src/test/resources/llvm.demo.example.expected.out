Data race on array element #21676936: \{\{\{(.*)---->  at <id:4865354;fn:Thread1;file:demo\/example.c;line:10>(.*)---->  at <id:4865467;fn:main;file:demo\/example.c;line:18>(.*)\}\}\}

